CONTRIBUTING.md

```diff
@@ -109,11 +116,17 @@ A good rule of thumb for the commit message is to have a present tense verb, fol
 $ git push -u origin <branch_name>  
 ```  
 - Here is how your branch will look.  
-![br](https://github.com/kanishkaa24/Community-Website/blob/main/3.png)  
+![br](https://github.com/LeeRenJie/Community-Website/blob/main/assets/cont-img/br.png)  
   
 ## Step 6 : Pull Request  🎣
```

Comment:
You missed Step 4 and 5

Reply:
There were no images initially so I thought it was unnecessary. Should I add?

Reply:
Done with step 4 & 5! Please review thanks!